### PR TITLE
Fix/automation locators

### PR DESCRIPTION
**Pull request by TiManGames**
Fixed an issue which prevents the `test-id` locators from showing up in the HTML.
Also updated the attribute values for some of the components.

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx [hunk 1]
```diff
@@ -133,13 +133,14 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
                     </b>
                 </Typography>
                 <Autocomplete
-                    test-id='currentQuarantineAddress'
+                    test-id='currentQuarantineCity'
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:136]: Why not testId

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx [hunk 2]
```diff
@@ -157,21 +158,23 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
...
-                        selectedStreet && updateIsolationAddress(ClinicalDetailsFields.ISOLATION_STREET, selectedStreet.id)}
+                        selectedStreet && updateIsolationAddress(ClinicalDetailsFields.ISOLATION_STREET, selectedStreet.id)
+                    }
                     }
                     onInputChange={(event, selectedStreetName) => {
                         setIsolationStreetName(selectedStreetName);
                     }}
                     renderInput={(params) =>
                         <CircleTextField
+                            test-id='currentQuarantineStreet'
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:169]: Same

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx [hunk 3]
```diff
@@ -157,21 +158,23 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
...
                             {...params}
                             placeholder='רחוב'
                             className={classes.textField}
                         />
                     }
                 />
                 <CircleTextField
-                    test-id='currentQuarantineAddress'
+                    test-id='currentQuarantineHomeNumber'
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:177]: Same

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx (hunk 1 above)
Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:136]: `testId` is a name of a paramter_, while `test-id` is the name of the actual HTML attribute.
Some components need to pass a parameter to sub-components and some don't.

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx (hunk 2 above)
Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:169]: `testId` is a name of a paramter_, while `test-id` is the name of the actual HTML attribute.
Some components need to pass a parameter to sub-components and some don't.

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx (hunk 3 above)
Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:177]: `testId` is a name of a paramter_, while `test-id` is the name of the actual HTML attribute.
Some components need to pass a parameter to sub-components and some don't.

**Review by RoeiRom — APPROVED** [review]

**Review by daniel7764 — APPROVED** [review]

**Pull request merged**